core/server/data/schema/schema.js

```diff
@@ -712,24 +731,6 @@ module.exports = {
         },
         value: {type: 'text', maxlength: 65535, nullable: true}
     },
-    newsletters: {
```

Comment:
I guess this has to move so that the table exists when the post table references it for the foreign key?

Reply:
Exactly, I did update the commit to mention this but forgot to update the PR!